R/module_teal_data.R

```diff
@@ -67,19 +66,7 @@ srv_teal_data <- function(id,
     } else {
       data_module$server(id = "data")
     }
-
-    data_validated <- srv_validate_reactive_teal_data(
-      id = "validate",
-      data = data_out,
-      modules = modules,
-      validate_shiny_silent_error = validate_shiny_silent_error
-    )
-
-    .fallback_on_failure(
-      this = data_validated,
-      that = data,
-      label = sprintf("Data element '%s' for module '%s'", id, modules$label)
-    )
```

Comment:
By removing this teal_transform_module loses warning/error capabilities.  `module_teal_data` is used to validate both `teal_data_module` and `teal_transform_module`